cisco_secure_firewall/README.md

```diff
@@ -0,0 +1,166 @@
+## Overview
+
+[Cisco Secure Firewall Threat Defense (FTD)][5] is a threat-focused, next-gen firewall (NGFW) with unified management. It provides advanced threat protection before, during, and after attacks.The [Cisco Secure Firewall Management Center (FMC)][7] is the centralized event and policy manager for Cisco Secure Firewall Threat Defense (FTD), both on-premises and virtual
+
+This integration enrich and ingests the following logs from Cisco Secure FTD via Cisco Secure FMC:
+- User Authentication Logs
+- SNMP Logs
+- Failover Logs
+- Transparent Firewall Logs
+- Threat Detection Logs
+- Security Events
+- IP Stack Logs
+- Application Firewall Logs
+- Identity-based Firewall Logs
+- Command Interface Logs
+- OSPF Rotuing Logs
+- RIP Routing Logs
+- Resource Manager Logs
+- VPN Failover Logs
+- Intrusion Protection System Logs
+- Dynamic Access Policies
+- IP Address Assignment
+
+Visualize detailed insights into snmp requests, identity based firewall logs, real time threat analysis, security detection and observation, compliance monitoring through the out-of-the-box dashboards.
+
+## Setup
+
+### Installation
+
+To install the Cisco Secure Firewall integration, run the following Agent installation command and the steps below. For more information, see the [Integration Management][6] documentation.
+
+**Note**: This step is not necessary for Agent version >= 7.52.0.
+
+Linux command
+  ```shell
+  sudo -u dd-agent -- datadog-agent integration install datadog-cisco_secure_firewall==1.0.0
+  ```
+
+### Configuration
+
+#### Cisco Secure Firewall
+
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in `datadog.yaml`:
+    ```yaml
+    logs_enabled: true
+    ```
+
+2. Add this configuration block to your `cisco_secure_firewall.d/conf.yaml` file to start collecting your Cisco Secure Firewall logs.
+
+    See the [sample cisco_secure_firewall.d/conf.yaml][6] for available configuration options.
+
+   ```yaml
+    logs:
+      - type: tcp/udp
+        port: <PORT>
+        service: cisco-secure-firewall
+        source: cisco-secure-firewall
+   ```
```

Comment:
Can you indent this code block one or two spaces to the right?

Reply:
Done